crates/hyperswitch_connectors/src/utils.rs

```diff
@@ -64,6 +66,11 @@ pub(crate) fn to_currency_base_unit_with_zero_decimal_check(
         .change_context(errors::ConnectorError::RequestEncodingFailed)
 }
 
+pub(crate) fn get_timestamp_in_milliseconds(datetime: &PrimitiveDateTime) -> i64 {
+    let utc_datetime = datetime.assume_utc();
+    utc_datetime.unix_timestamp() * 1000
```

Comment:
One other way to obtain milliseconds is to obtain the timestamp in nanoseconds and then divide by 10^6.

```suggestion
    utc_datetime.unix_timestamp_nanos() / 1_000_000
```

This would not return `i64` though, so there's that.